app/assets/javascripts/govuk_publishing_components/components/search-with-autocomplete.js

```diff
@@ -0,0 +1,99 @@
+/* global accessibleAutocomplete, fetch */
+//= require accessible-autocomplete/dist/accessible-autocomplete.min.js
+
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  class SearchWithAutocomplete {
```

Comment:
I think it's used where there might be another component in an application and the module names could collide. In this case it's probably sensible because we have other autocomplete components, although they're all named autocomplete.

Reply:
Yeah – I saw there was a mix of both approaches but agree in this case it makes sense to go the prefix route!